core/server/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,6 +39,7 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
+      LOG.error("Internal Alluxio error when running rpc", e);
```

Comment:
Should this be debug-level? `AlluxioException` usually means there was an error by the client, not Alluxio.